### PR TITLE
Add custom function that run when parameter is set

### PR DESCRIPTION
**Pull request by John-Holt-Tessella**
Also:
 Change _JawsAxisPVWrapper to JawsAxisPVWrapper because it is used as a type in another module

For:

https://github.com/ISISComputingGroup/IBEX/issues/5532

### CODE REVIEW

File: ReflectometryServer/parameters.py [hunk 1]
```diff
@@ -860,7 +916,8 @@ def _initialise_sp_from_file(self):
 
     def _initialise_sp_from_motor(self, _):
         STATUS_MANAGER.update_error_log("Optional parameter, {}, was asked up init from motor".format(self.name))
-        STATUS_MANAGER.update_active_problems("Optional Parameter updating from motor")
+        STATUS_MANAGER.update_active_problems(ProblemInfo("Optional Parameter updating from motor", self.name,
+                                                          Severity.MAJOR_ALARM))
```
Comment (DominicOram) [inline ReflectometryServer/parameters.py:920]: Is this really major? Seems informational?

**Review by DominicOram — CHANGES_REQUESTED** [review]
Works perfectly, good tests. Minor comment in code. Also do we have a note somewhere to get the actual wiring table change code on to inter?

File: ReflectometryServer/parameters.py (hunk 1 above)
Comment (John-Holt-Tessella) [reply ReflectometryServer/parameters.py:922]: Have added comment and will add note for wiring tables to snags list because we need user input

**Review by DominicOram — APPROVED** [review]
Great, thanks John

**Pull request merged**